README.md

```diff
@@ -25,3 +25,42 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+$$
+T(n) = 
+\begin{cases} 
+\(1) & \text{if } n \le 1, \\
+3T\left(\frac{n}{3}\right) + (n) + O(1) & \text{if } n > 1
```

Comment:
Where is the n coming from here? Are you doing a linear amount of work to combine the partial results?

Reply:
Ahh, I think I got it this time, we are not recursively merging the arrays back together here like we are in mergesort, so instead of n, we have O(1), which yield O(n) over each level of recursion.